releasenotes/notes/remove_probability_distributions-d30bd77f0f2b9570.yaml

```
@@ -0,0 +1,8 @@
+---
+upgrade:
+  - |
+    The module ``qiskit.circuit.library.probability_distributions`` has been removed and no longer exist as per the deprecation notice from 0.17.0
+    (released Apr 1, 2021) via #6080 <https://github.com/Qiskit/qiskit-terra/issues/6080>`__. 
+    The affected classes are ``UniformDistribution``,
+    ``NormalDistribution``, and ``LogNormalDistribution``. They are all moved to qiskit-finance, into the circuit library module
+    (`qiskit_finance.circuit.library.probability_distributions`).
```

Should this be in double quotations as it's code?
```suggestion
    (``qiskit_finance.circuit.library.probability_distributions``).
```